deployments/helm/hephaestus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: hephaestus
-version: 0.0.0
+version: 0.0.1
```

Comment:
This isn't necessary, it's auto-filled in at release time to the correct tag

Reply:
got it, change reverted